llm/validation_test.go

```diff
@@ -0,0 +1,180 @@
+package llm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Possible to use the `testing` library vs testify (trying to move away from it where possible)

Reply:
Done, sorry I missed that one!